src/main/java/Duke.java

```diff
@@ -47,50 +48,52 @@ public static void addTodo(String todo) {
         currentTask++;
         System.out.println(line);
         System.out.println("\tGot it. I've added this task:");
-		System.out.println("\t   [" + t.getType() + "][" + t.getStatusIcon() + "] " + t.getDescription());
-		System.out.println("\tNow you have " + Integer.toString(currentTask) + " tasks in the list.");
+        System.out.println("\t   [" + t.getType() + "][" + t.getStatusIcon() + "] " + t.getDescription());
+        System.out.println("\tNow you have " + Integer.toString(currentTask) + " tasks in the list.");
         System.out.println(line + "\n");
     }
 
-	public static void addEvent(String event) {
-		try {
-			String description = event.split("/at")[0].strip();
-			String date = event.split("/at")[1].strip();
-			Event t = new Event(description, date);
-			tasks[currentTask] = t;
-			currentTask++;
-			System.out.println(line);
-			System.out.println("\tGot it. I've added this task:");
-			System.out.println("\t   [" + t.getType() + "][" + t.getStatusIcon() + "] " + t.getDescription() + " " + t.getDate());
-			System.out.println("\tNow you have " + Integer.toString(currentTask) + " tasks in the list.");
+    public static void addEvent(String event) {
+        try {
+            String description = event.split("/at")[0].strip();
+            String date = event.split("/at")[1].strip();
+            Event t = new Event(description, date);
+            tasks[currentTask] = t;
+            currentTask++;
+            System.out.println(line);
+            System.out.println("\tGot it. I've added this task:");
+            System.out.println("\t   [" + t.getType() + "][" + t.getStatusIcon() + "] "
+                    + t.getDescription() + " " + t.getDate());
+            System.out.println("\tNow you have " + Integer.toString(currentTask) + " tasks in the list.");
     	    System.out.println(line + "\n");
-		}
-		catch (ArrayIndexOutOfBoundsException e) {
-			System.out.println(line);
-			System.out.println("\tPlease specify event date.");
-			System.out.println(line + "\n");
-		}
-	}
+        }
+        catch (ArrayIndexOutOfBoundsException e) {
```

Comment:
maybe you could bring the catch statement to the } bracket of your try statement ( like "} catch (...) {" ) I think that is a violation that you could change:)